Release/include/cpprest/http_client.h

```diff
@@ -279,6 +281,36 @@ class http_client_config
     void set_buffer_request(bool buffer_request) { m_buffer_request = buffer_request; }
 #endif
 
+    /// <summary>
+    /// Get the maximum number of redirects to follow automatically.
+    /// A value of 0 indicates that no automatic redirection is performed.
+    /// </summary>
+    /// <returns>The maximum number of redirects to follow automatically.</returns>
+    size_t max_redirects() const { return m_max_redirects; }
```

Comment:
It seems like these members should only exist for the asio backend because the winhttp backend won't honor the setting. (Alternately we shouldn't provide the knob?)

Reply:
Knowing it's likely to get merged/released, I'll happily work on these settings behaving similarly for the WinHTTP backend, or propose a lowest common denominator of a simple `bool` enable / disable.